test/test_selector.js

```diff
@@ -247,10 +247,12 @@ suite('selector', () => {
       a => a
     )
     assert.equal(selector({ a: 1 }), 1)
-    assert.equal(selector({ a: 2 }), 1) // yes, really true
+    // still returns `1` since `typeof state` is still "object"
+    assert.equal(selector({ a: 2 }), 1)
+    assert.equal(selector.recomputations(), 1)
+    // again returns `1` since `typeof state` is still "object"
+    assert.equal(selector({ a: 'A' }), 1)
```

Comment:
its worth highlighting that this is a subtle change in behavior and could break dependent codebases.  So bumping a version number seems to be in order